lib/statsd/instrument.rb

```diff
@@ -191,6 +193,7 @@ def statsd_count_if(method, name, *metric_options)
           ensure
             if truthiness
               metric_name = StatsD::Instrument.generate_metric_name(name, self, *args)
+              # TODO: is this a bug, because 1 is missing?
```

Comment:
It is probably a bug, given that it is inconsistent with `statsd_count_success` in this way.  That means that the `value` is being provided in the third positional argument to `statsd_count_if`, but in `statsd_count_success` it is the sample rate.

Reply:
Another good reason to not use positional arguments :)

I fixed this in https://github.com/Shopify/statsd-instrument/pull/182/